com.github.jeromerobert.pdfarranger.yaml

```diff
@@ -0,0 +1,73 @@
+---
+id: com.github.jeromerobert.pdfarranger
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: pdfarranger
+
+# These renames will be obsolete with 
+# https://github.com/jeromerobert/pdfarranger/issues/158
+# Currently known bug: Missing icon in about dialog (Will be fixed by #158.)
+rename-icon: pdfarranger
+rename-desktop-file: pdfarranger.desktop
+rename-appdata-file: pdfarranger.appdata.xml
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=home
+
+modules:
+  - shared-modules/intltool/intltool-0.51.json
+  - shared-modules/openjpeg/openjpeg.json # dependency of poppler
+
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DENABLE_UTILS=OFF
+      - -DENABLE_CPP=OFF
+      - -DENABLE_QT5=OFF
+    sources:
+      - url: https://poppler.freedesktop.org/poppler-0.86.1.tar.xz
+        sha256: af630a277c8e194c31339c5446241834aed6ed3d4b4dc7080311e51c66257f6c
+        type: archive
+
+  - name: python-distutils-extra
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} .
+    sources:
+      - url: https://launchpad.net/python-distutils-extra/trunk/2.39/+download/python-distutils-extra-2.39.tar.gz
+        sha256: 723f24f4d65fc8d99b33a002fbbb3771d4cc9d664c97085bf37f3997ae8063af
+        type: archive
+
+  - name: pikepdf
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} lxml-*.whl pikepdf-*.whl
+    sources:
+      - url: https://files.pythonhosted.org/packages/38/e8/9cce836a4b910e3928125d5566592b6531a791cfd7e90636d5104c497c65/pikepdf-1.10.3-cp37-cp37m-manylinux2010_x86_64.whl
+        sha256: 9f05a5f74ba58a2ae050da1bca44a0663f5ef94f42255225f44e38d0c60439af
+        type: file
+      - url: https://files.pythonhosted.org/packages/85/9e/93e2c3af278c7c8b6826666bbcb145af2829bd761c3b329e51cd6343836c/lxml-4.5.0-cp37-cp37m-manylinux1_x86_64.whl
+        sha256: fd52e796fee7171c4361d441796b64df1acfceb51f29e545e812f16d023c4bbc
+        type: file
+
+  - name: pdfarranger
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} .
+    sources:
+      - url: https://github.com/jeromerobert/pdfarranger
+        tag: 1.4.2
+        commit: b1622c9244b3c1ec005bdd458864bf580c7dcd68
+        type: git
+      - path: appdata.patch # already in upstream/master but not in 1.4.2
+        type: patch
+
+cleanup:
```

Comment:
Nitpick: this should go before modules

Reply:
Is there an ordering guide or a reference json/yaml somewhere? Cleanup should go before modules but after finish-args?

Reply:
well, there a bunch of manifests on github.com/flathub, but we don't have a set of rules written nowhere